src/dmtcp_coordinator.cpp

```diff
@@ -423,6 +423,7 @@ void DmtcpCoordinator::updateMinimumState(WorkerState oldState)
     if (exitAfterCkpt || exitAfterCkptOnce) {
       JNOTE("Checkpoint Done. Killing all peers.");
       broadcastMessage(DMT_KILL_PEER);
+      JTIMER_STOP ( checkpoint );
```

Comment:
Let's move this before `broadcastMessage`. That could take a long time if there are lot of peers.